rkt/fetch.go

```diff
@@ -85,9 +88,38 @@ func runFetch(args []string) (exit int) {
 
 // fetchImage will take an image as either a URL or a name string and import it
 // into the store if found.  If discover is true meta-discovery is enabled.
-func fetchImage(img string, ds *cas.Store, ks *keystore.Keystore, discover bool) (string, error) {
+// if asc is not "", it must exist as a local file and will be used as the signature file for verification unless verification is disabled.
+func fetchImage(img string, asc string, ds *cas.Store, ks *keystore.Keystore, discover bool) (string, error) {
+	var (
+		ascFile *os.File
+		err     error
+	)
+	if asc != "" && ks != nil {
```

Comment:
hrm, if asc is supplied but keystore is nil, wonder if we should report this somewhere?